### DISCUSSION TITLE
笔记优化步骤，参考Word，可以划词做高亮、加粗、调大小

### DISCUSSION THREAD

**Discussion by 202315903** [windingwind/zotero-better-notes#494] [Ideas]
可以划词做高亮、加粗、调大小

**Comment by youngfish42** [comment]
你说的这些功能，除了字体大小不好调整外（字体大小调整应该可以通过html语言来实现），其他都原生支持的

![图片](https://github.com/windingwind/zotero-better-notes/assets/38998412/733976ea-fd21-46a6-8e59-25256303ac8a)

**Comment by 202315903** [comment]
感谢你的回复。在word做笔记，可以划词一步高亮、设置样式等。


在better note，需要两步。







------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Yuwen ***@***.***&gt;; 
发送时间: 2023年6月27日(星期二) 晚上8:05
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [windingwind/zotero-better-notes] 笔记优化步骤，参考Word，可以划词做高亮、加粗、调大小 (Discussion #494)





 
你说的这些功能，除了字体大小不好调整外（字体大小调整应该可以通过html语言来实现），其他都原生支持的

**Reply by youngfish42** [reply]
哦哦 我有点理解你的需求了

有些操作是可以通过快捷键实现的，比如加粗是 ctrl+B ，斜体是 ctrl+ I （高亮我不知道要按啥），熟练后也很方便。
不过你想就按一个键就改变格式，目前确实在支持markdown的阅读器中比较少见。

进一步的，markdown阅读器中也很少见word的格式刷功能，确实某些程度上便利性要差一些。

**Comment by 202315903** [comment]
好的，感谢，确实，都忘记快捷键的通用了，感谢






------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Yuwen ***@***.***&gt;; 
发送时间: 2023年6月27日(星期二) 晚上8:16
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [windingwind/zotero-better-notes] 笔记优化步骤，参考Word，可以划词做高亮、加粗、调大小 (Discussion #494)





 
哦哦 我有点理解你的需求了
 
有些操作是可以通过快捷键实现的，比如加粗是 ctrl+B ，斜体是 ctrl+ I （高亮我不知道要按啥），熟练后也很方便。
 不过你想就按一个键就改变格式，目前确实在支持markdown的阅读器中比较少见。
 
进一步的，markdown阅读器中也很少见word的格式刷功能，确实某些程度上便利性要差一些。
